### PR TITLE
Split docker-compose template into couchdb + services

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

medic/cht-core#7697

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: tests/page-objects/display/languages.po.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const helper = require('../../helper');
 const utils = require('../../utils');
+const { element } = require('protractor');
```
Comment (garethbowen) [inline tests/page-objects/display/languages.po.js:3]: Why is this needed now?

File: tests/utils.js [hunk 2]
```diff
@@ -42,8 +42,10 @@ let e2eDebug;
 const hasModal = () => element(by.css('#update-available')).isPresent();
 const COUCH_USER_ID_PREFIX = 'org.couchdb.user:';
 
-const COMPOSE_FILE = path.resolve(__dirname, 'cht-compose-test.yml');
-const COMPOSE_FILE_TEMPLATE = path.resolve(__dirname, '..', 'scripts', 'build', 'cht-compose.yml.template');
+const SERVICES_COMPOSE_FILE = path.resolve(__dirname, 'cht-services-test.yml');
+const COUCHDB_COMPOSE_FILE = path.resolve(__dirname, 'cht-couchdb-test.yml');
+const SERVICES_COMPOSE_FILE_TEMPLATE = path.resolve(__dirname, '..', 'scripts', 'build', 'cht-services.yml.template');
+const COUCHDB_COMPOSE_FILE_TEMPLATE = path.resolve(__dirname, '..', 'scripts', 'build', 'cht-couchdb.yml.template');
```
Comment (garethbowen) [inline tests/utils.js:48]: NIT: This feels repetitive. Can we just have an array of filenames, eg:

```js
const COMPOSE_FILES = [ 'cht-services', 'cht-couchdb' ];
```

Then we can loop to generate the paths.

**Review by garethbowen — CHANGES_REQUESTED** [review]
Awesome.

One naming suggestion and two nitpicks...

File: scripts/build/cht-services.yml.template [hunk 3]
```diff
@@ -0,0 +1,46 @@
+version: '3.9'
```
[the remark this answers is not in the document]
Comment (dianabarsan) [reply scripts/build/cht-services.yml.template:1]: sure :D

File: tests/page-objects/display/languages.po.js (hunk 1 above)
Comment (dianabarsan) [reply tests/page-objects/display/languages.po.js:3]: Oh, I think my IDE autocompleted this and I failed to notice, so sorry!

File: tests/utils.js (hunk 2 above)
Comment (dianabarsan) [reply tests/utils.js]: Done.

**Review by garethbowen — APPROVED** [review]
Awesome. Thanks!
